v2x_ros_driver/src/v2x_radio_client.cpp

```diff
@@ -208,14 +184,14 @@ void V2XRadioClient::process(const std::shared_ptr<const std::vector<uint8_t>>&
     }
 }
 
-bool V2XRadioClient::isValidMsgSize(const std::vector<uint8_t> msg_vec, size_t start_index, size_t end_index, const std::vector<uint8_t> entry)
+bool V2XRadioClient::isValidMsgSize(const std::vector<uint8_t> msg_vec, size_t start_index, const std::vector<uint8_t> entry)
```

Comment:
Missed this in the previous PR, but can we update isValidMsgSiz() to pass `msg_vec` and `entry` by const reference?